lib/surface_site_web/live/slots.ex

```diff
@@ -49,72 +53,71 @@ defmodule SurfaceSiteWeb.Slots do
-                  </#Raw>
+                  <#Code
+                    language="elixir"
+                    module={{ SurfaceSiteWeb.Slots.SlotFallbackExample }}
+                    line_range={{ 4..21 }}
+                    show_line_numbers
+                    selected_lines="5, 11-13"
+                  />
                 </footer>
               </div>
 
               <#Markdown>
-                Now, if the user tries to use a `Hero` without defining any content, a proper
-                `missing required slot "default"` error will be raised at compile-time.
+                By defining any children inside `<slot>...</slot>`, that content becomes the default content.
+                Note that we removed the `required` option in the `slot` declaration. If we had not done so,
+                a warning would inform the user that fallback content has no effect, thanks to the compiler!
```

Comment:
```suggestion
                a warning would inform the user that the fallback content would have no effect, thanks to the compiler!
```

Reply:
For this, should we say `a warning would inform you` instead of `inform the user`?